src/rust/engine/process_execution/Cargo.toml

```diff
@@ -43,18 +43,20 @@ lazy_static = "1"
 parking_lot = "0.12"
 itertools = "0.10"
 serde = "1.0.136"
-bincode = "1.3.3"
 async-oncecell = "0.2"
+bincode = "1.3.3"
 once_cell = "1.15"
-rand = "0.8"
+os_pipe = "1.0"
```

Comment:
This might have been a merge conflict with a branch that I landed and then reverted: sorry!
```suggestion
```